scene/gui/graph_edit.cpp

```diff
@@ -1292,6 +1299,7 @@ void GraphEdit::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("duplicate_nodes_request"));
 	ADD_SIGNAL(MethodInfo("node_selected", PropertyInfo(Variant::OBJECT, "node", PROPERTY_HINT_RESOURCE_TYPE, "Node")));
 	ADD_SIGNAL(MethodInfo("connection_to_empty", PropertyInfo(Variant::STRING, "from"), PropertyInfo(Variant::INT, "from_slot"), PropertyInfo(Variant::VECTOR2, "release_position")));
+	ADD_SIGNAL(MethodInfo("connection_from_empty", PropertyInfo(Variant::STRING, "to"), PropertyInfo(Variant::INT, "to_slot"), PropertyInfo(Variant::VECTOR2, "release_position")));
```

Comment:
Is this still a "release_position" in this case, or would it be the "start_position"?

Reply:
Well, it will be valid in both cases because user releases the mouse in both cases. If I rename it to "start_position" then by same logic "connection_to_empty" should have "end_position" ?

Reply:
I mean, does this variable give the position of the mouse release in both cases? If so, its name is correct, but it doesn't really seem useful in the `from_empty` method? One would likely know where the "empty" start position was, and not where the mouse was released on a slot.

Reply:
Hmm.. yes, the release_position can be used in both cases. They are similar to each other - the difference is just the direction of ports (input or output).

 In the case of visual shader editor there is already a call from `_show_members_dialog` by `Input::get_singleton()->get_mouse_position()` so its not used directly.